src/Driver/PDO/Statement.php

```diff
@@ -145,6 +145,8 @@ public function execute($params = null): ResultInterface
      * @param int $type Parameter type
      *
      * @throws ExceptionInterface
+     *
+     * @return value-of<\Doctrine\DBAL\Driver\PDO\Statement::PARAM_TYPE_MAP>
```

Comment:
I think we can keep this one a bit simpler.

```suggestion
     * @return PDO::PARAM_*
```

Reply:
If filled in `@return PDO::PARAM_*` will get error:

```bash
------ ----------------------------------------------------------------------------------------------------------------------------------- 
  Line   Driver/PDO/Statement.php                                                                                                           
 ------ ----------------------------------------------------------------------------------------------------------------------------------- 
  151    Method Doctrine\DBAL\Driver\PDO\Statement::convertParamType() never returns 1073741824 so it can be removed from the return type.  
  151    Method Doctrine\DBAL\Driver\PDO\Statement::convertParamType() never returns 2147483648 so it can be removed from the return type.  
  151    Method Doctrine\DBAL\Driver\PDO\Statement::convertParamType() never returns 4 so it can be removed from the return type.           
  151    Method Doctrine\DBAL\Driver\PDO\Statement::convertParamType() never returns 536870912 so it can be removed from the return type.   
  151    Method Doctrine\DBAL\Driver\PDO\Statement::convertParamType() never returns 6 so it can be removed from the return type.           
 ------ -----------------------------------------------------------------------------------------------------------------------------------
```

This is because the return type of `convertParamType()` is the value of `self::PARAM_TYPE_MAP`, which does not completely override `PDO::RARAM_*`

---

* PDO::PARAM_* is `0|1|2|3|4|5|6|536870912|1073741824|2147483648`
* convertParamType() will return `0|1|2|3|5`

Reply:
Then ignore that error please. We don't need that level of granularity here. The contract of the `convertParamType()` method is that it returns a value that can be funneled into `bindValue()` or `bindParam()` and that exactly what `PDO::RARAM_*` says.